admin/modules/config/settings.php

```diff
@@ -1186,7 +1186,20 @@
 			}
 			else if($type[0] == "numeric")
 			{
-				$setting_code = $form->generate_numeric_field($element_name, $setting['value'], array('id' => $element_id));
+				$field_options = array('id' => $element_id);
+				if (count($type)>1)
```

Comment:
As already said: it's just a copy of the old PR without any work and is still WIP...